### ISSUE TITLE
Finding Things - Exercise 'Tracking a Species' only has the help function for Mac listed

### ISSUE THREAD

**Issue by Elisa-on-GitHub** [swcarpentry/shell-novice#1314]
In the episode 'Finding Things' the ['Tracking a Species' exercise](https://swcarpentry.github.io/shell-novice/07-find/index.html#tracking-a-species) advises a learner to "use `man grep` to look for how to grep text recursively in a directory and `man cut` to select more than one field in a line".
This is only the help command for Mac. For people working on a Windows machine with Git Bash, the command would be `grep --help` and `cut --help` respectively.

My suggestion: change the wording in this exercise to:
"... use `man grep` (Mac) or `grep --help` (Git on Windows) to look for how to grep text recursively in a directory and `man cut` (Mac) or `cut --help` (Git on Windows) to select more than one field in a line"

I don't know what the help function is on Linux, because I don't use Linux personally and the [Getting Help section](https://swcarpentry.github.io/shell-novice/02-filedir/index.html#getting-help) in the first episode does not specify it. I suppose one of the commands for Mac or Windows will work. Is there a Linux user who knows? Thank you in advance 🙏

**Comment by gcapes** [comment]
This is a good point. Both `man` and `--help` are available for Linux, so that would round out your proposed changes.

**Comment by Elisa-on-GitHub** [comment]
Thank you for your feedback. Do you think this warrants a PR? I would be happy to submit it!

Update: I added issue #1315, which has been followed up.

**Comment by gcapes** [comment]
Yes I think a PR would be useful to remind learners in this exercise that if they're using Git Bash, they won't have `man` available.
